packages/core/src/__tests__/DdSdkReactNative.test.tsx

```diff
@@ -664,6 +667,49 @@ describe('DdSdkReactNative', () => {
             );
         });
 
+        it('enables action mapping when initialize { action mapper enabled }', async () => {
+            // GIVEN
+            const fakeAppId = '1';
+            const fakeClientToken = '2';
+            const fakeEnvName = 'env';
+            const configuration = new DdSdkReactNativeConfiguration(
+                fakeClientToken,
+                fakeEnvName,
+                fakeAppId,
+                false,
+                false,
+                true
+            );
+            configuration.actionEventMapper = event => {
+                event.context = {
```

Comment:
I'll add the action name in the test! Since we apply the mapper before calling the native side we can safely allow the context modification at this stage (I understand it's harder to do on the native side as the context will be enriched by some other data like view id for instance)